environment-setup/ansible/TP2/roles/monitoring-service/templates/tp3-monitor-service.service.j2

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
This file and tp3-monitor.py.j2 do not seem to do any templating, we can move them into `monitoring-service/files` and just copy them verbatim.